commands/new.go

```diff
@@ -117,13 +119,24 @@ func NewSite(cmd *cobra.Command, args []string) {
 		jww.FATAL.Fatalln(err)
 	}
 
-	if x, _ := helpers.Exists(createpath, hugofs.SourceFs); x {
-		y, _ := helpers.IsDir(createpath, hugofs.SourceFs)
-		if z, _ := helpers.IsEmpty(createpath, hugofs.SourceFs); y && z {
-			jww.INFO.Println(createpath, "already exists and is empty")
-		} else {
+	if exists, _ := helpers.Exists(createpath, hugofs.SourceFs); exists {
+		if isDir, _ := helpers.IsDir(createpath, hugofs.SourceFs); !isDir {
+			jww.FATAL.Fatalln(createpath, "already exists but not a directory")
+		}
+
+		if isEmpty, _ := helpers.IsEmpty(createpath, hugofs.SourceFs); !isEmpty && !forceNew {
 			jww.FATAL.Fatalln(createpath, "already exists and is not empty")
 		}
+
+		if forceNew {
+			if err = os.RemoveAll(createpath); err != nil {
```

Comment:
The flag help text says "create even if destination is not empty"". It says nothing about deleting stuff. And rightfully so.